synology/start_tunnel_zabbix_proxy_to_server.sh

```diff
@@ -27,17 +35,20 @@ echo "Starting tunnel to Server..."
   -L 10052:localhost:10051 \
   -i "${priv_key}" \
   -o ProxyCommand="ssh -W %h:%p tunneller@sshrelay.ocb.msf.org" \
-  -o "ExitOnForwardFailure=yes" \
-  -o "ServerAliveInterval=10" \
-  -o "ServerAliveCountMax=5" \
-  -o "ConnectTimeout=360" \
-  -o "UpdateHostKeys=yes" \
-  -o "StrictHostKeyChecking=yes" \
-  -o "IdentitiesOnly=yes" \
-  -o "Compression=yes" \
-  -o "ControlMaster=no" \
+                             -o "ExitOnForwardFailure=yes" \
+                             -o "ServerAliveInterval=10" \
+                             -o "ServerAliveCountMax=5" \
+                             -o "ConnectTimeout=360" \
+                             -o "UpdateHostKeys=yes" \
+                             -o "StrictHostKeyChecking=yes" \
+                             -o "IdentitiesOnly=yes" \
+                             -o "Compression=yes" \
+                             -o "ControlMaster=no" \
+                             -o "UserKnownHostsFile=${known_hosts_file_server}" \
+                             -o "LogLevel=ERROR" \
+                             -o "AddKeysToAgent=yes" \
```

Comment:
It's not sufficient to just indent these lines, bash does not care about indentation, look closely at the `"` in the script I linked to.